django/core/management/commands/migrate.py

```diff
@@ -141,14 +141,17 @@ def handle(self, *args, **options):
                     raise CommandError("Cannot find a migration matching '%s' from app '%s'." % (
                         migration_name, app_label))
                 target = (app_label, migration.name)
-                # Partially applied squashed migrations are not included in the
-                # graph, use the last replacement instead.
-                if (
-                    target not in executor.loader.graph.nodes and
-                    target in executor.loader.replacements
-                ):
-                    incomplete_migration = executor.loader.replacements[target]
-                    target = incomplete_migration.replaces[-1]
+                if target not in executor.loader.graph.nodes:
+                    # Partially applied squashed migrations are not included in the
+                    # graph, use the last replacement instead.
+                    if target in executor.loader.replacements:
+                        incomplete_migration = executor.loader.replacements[target]
+                        target = incomplete_migration.replaces[-1]
+                    # Reload the graph without replacements to support migrating
+                    # backward to a replaced migration.
+                    else:
+                        executor.loader.replace_migrations = False
+                        executor.loader.build_graph()
```

Comment:
Changes in the `migrate` command are unnecessary when you added a similar fix to the executor.